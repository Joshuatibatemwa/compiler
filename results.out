_______First Test_____
Usage: my-project [flags] [args] 
 Available flags:
  -length Prints length of all arguments
  -help  Display this list of options
  --help  Display this list of options
_______Second Test_____
One
for
each
line
in
terminal
_______Third Test_____
3 
4 
3 
4 
2 
5 
5 
_______Fourth Test_____
I
get
nothing
from
this

