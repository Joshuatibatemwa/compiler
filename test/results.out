===== test1 =====
6
===== test2 =====
[(,let,max,=,(,fun,x,->,(,if,(,x,<=,5,),then,6,else,5,),),in,(,max,(,4,),),)]
===== test3 =====
let max = fun x -> if x <= 5 then 6 else 5 in max (4)
===== test4 =====
-> let max = fun x -> if x <= 5 then 6 else 5 in max (4)
-> fun x -> if x <= 5 then 6 else 5 (4)
-> if 4 <= 5 then 6 else 5
-> if true then 6 else 5
-> 6
===== test5 =====
0
===== test6 =====
[(,3,/,(,2,+,(,5,-,(,6,*,2,),),),)]
===== test7 =====
3 / 2 + 5 - 6 * 2
===== test8 =====
-> 3 / 2 + 5 - 6 * 2
-> 3 / 2 + 5 - 12
-> 3 / 2 + -7
-> 3 / -5
-> 0
===== test9 =====
38
===== test10 =====
[(,if,(,5,<=,3,),then,(,100,-,5,),else,(,33,+,5,),)]
===== test11 =====
if 5 <= 3 then 100 - 5 else 33 + 5
===== test12 =====
-> if 5 <= 3 then 100 - 5 else 33 + 5
-> if false then 100 - 5 else 33 + 5
-> 38
===== test13 =====
1
===== test14 =====
[(,if,(,3,<=,4,),then,(,if,true,then,1,else,2,),else,(,2,+,3,),)]
===== test15 =====
if 3 <= 4 then if true then 1 else 2 else 2 + 3
===== test16 =====
-> if 3 <= 4 then if true then 1 else 2 else 2 + 3
-> if true then if true then 1 else 2 else 2 + 3
-> 1
===== test17 =====
6
===== test18 =====
[(,let,max,=,(,fun,x,->,(,if,(,x,<=,5,),then,6,else,5,),),in,(,max,(,4,),),)]
===== test19 =====
let max = fun x -> if x <= 5 then 6 else 5 in max (4)
===== test20 =====
-> let max = fun x -> if x <= 5 then 6 else 5 in max (4)
-> fun x -> if x <= 5 then 6 else 5 (4)
-> if 4 <= 5 then 6 else 5
-> if true then 6 else 5
-> 6
